src/components/ModelingMachineProvider.tsx

```diff
@@ -41,7 +41,7 @@ import {
   canSweepSelection,
   handleSelectionBatch,
   isSelectionLastLine,
-  isRangeInbetweenCharacters,
+  isRangeInbetweenCharacters as isRangeBetweenCharacters,
```

Comment:
C: Probably just trying to keep the unrelated changes down here yeah?

Reply:
nah, I used symbol rename in vscode and didn't check, it sometimes does dodgy stuff like this.

Thanks for picking this up.